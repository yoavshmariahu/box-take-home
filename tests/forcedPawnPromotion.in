k a1
K e5
p a4
P e2

[G S B R]
[r b s g]

move a4 a5
move e2 e1
