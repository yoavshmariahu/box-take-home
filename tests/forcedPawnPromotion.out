UPPER player action: move e2 e1
5 |+p|__|__|__| K|
4 |__|__|__|__|__|
3 |__|__|__|__|__|
2 |__|__|__|__|__|
1 | k|__|__|__|+P|
    a  b  c  d  e

Captures UPPER: G S B R
Captures lower: r b s g

lower> 
